### PR TITLE
Rl 26

### PR DESCRIPTION
**Pull request by Squidtito**
"PlayerTest" room added alongside with a "Health" variable for the player.

### CODE REVIEW

File: scripts/player.gd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 extends KinematicBody2D
 var velocity:Vector2 = Vector2(0,0)
 var up:Vector2 = Vector2(0,-1)
-
+var Health : float = 100.0
```
Comment (keri4141) [inline scripts/player.gd]: can you have this as health instead of Health

File: scripts/player.gd [hunk 2]
```diff
@@ -1,7 +1,7 @@
 extends KinematicBody2D
 var velocity:Vector2 = Vector2(0,0)
```
Comment (keri4141) [inline scripts/player.gd]: can you also fix the spacing here between the type and variable name

var velocity : Vector2

File: scripts/player.gd [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 var up:Vector2 = Vector2(0,-1)
```
Comment (keri4141) [inline scripts/player.gd]: same for this

**Review by keri4141 — COMMENTED** [review]
need a few fixes regarding upper case and lower case, and spacing between the variable type

File: scripts/player.gd (hunk 1 above)
Comment (Squidtito) [reply scripts/player.gd]: Alright, Tree will also will have to make his health variable lowercase

**Comment by Squidtito** [comment]
Alright, I've fixed the following issues, I'm going to push them in a minute.

File: scripts/player.gd (hunk 1 above)
Comment (keri4141) [reply scripts/player.gd]: Yup this is correct :)

**Review by keri4141 — APPROVED** [review]

**Pull request merged**